lib/channels/app_socket.ex

```diff
@@ -78,7 +79,10 @@ defmodule ApplicationRunner.AppSocket do
             {:error, ErrorHelpers.translate_error(reason)}
 
           {:error, reason} ->
-            Logger.error(reason)
+            reason
+            |> Errors
```

Comment:
What is that ? Probably a fail :p
If not, It's the first time I see that in elixir.

Reply:
Totally !
My bad